.python-version-default

```diff
@@ -1 +1 @@
-3.13
+3.13.2
```

Comment:
```suggestion
3.13
```

Please don't – it's specifically named `-default`, such that people can have their own `.python-version` either by making a link or setting to a specific version like you would like